samples/word/50-document/manage-tracked-changes.yaml

```diff
@@ -0,0 +1,218 @@
+          await Word.run(async (context) => {
+            const body = context.document.body;
+            const trackedChanges = body.getTrackedChanges();
+            const trackedChange = trackedChanges.getFirst();
+            await context.sync();
+
+            const range = trackedChange.getRange();
+            range.load();
+            await context.sync();
+            
+            console.log("range.text: " + range.text);
+          });
+        }
+
+        async function acceptFirstTrackedChange() {
+          // Accepts the first tracked change.
+          await Word.run(async (context) => {
+            const body = context.document.body;
+            const trackedChanges = body.getTrackedChanges();
+            const trackedChange = trackedChanges.getFirst();
+            trackedChange.load();
+            await context.sync();
+
+            console.log("First tracked change:");
+            console.log(trackedChange);
+            trackedChange.accept();
+            console.log("Accepted the first tracked change");
+          });
+        }
+
+        async function rejectFirstTrackedChange() {
+          // Rejects the first tracked change.
+          await Word.run(async (context) => {
+            const body = context.document.body;
+            const trackedChanges = body.getTrackedChanges();
+            const trackedChange = trackedChanges.getFirst();
+            trackedChange.load();
+            await context.sync();
+
+            console.log("First tracked change:");
+            console.log(trackedChange);
+            trackedChange.reject();
+            console.log("Rejected the first tracked change");
+          });
+        }
+
+        async function setup() {
+          // Updates the text and sets the font color to red.
+          await Word.run(async (context) => {
+            context.document.changeTrackingMode = Word.ChangeTrackingMode.off;
+
+            context.document.body.insertText("AAA BBB CCC DDD EEE FFF", "Replace");
+
+            context.document.changeTrackingMode = Word.ChangeTrackingMode.trackAll;
+            context.document.body
+              .search("BBB")
+              .getFirst()
+              .insertText("WWW", "Replace");
+            context.document.body
+              .search("DDD ")
+              .getFirst()
+              .delete();
+            context.document.body
+              .search("FFF")
+              .getFirst()
+              .insertText("XXX ", "Start");
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
```

Comment:
Suggestion: Consider grouping similar operations together. For example:

- Get all tracked changes
- Get the next (second) tracked change
- Get the first tracked change's range
- Accept the first tracked change
- Reject the first tracked change

Choose the **Add sample text** button before testing the following options.
- Accept all tracked changes
- Reject all tracked changes

Reply:
Reorganized and ready for your review